webportal/src/app/job/job-view/job-detail-table.component.ejs

```diff
@@ -98,6 +99,11 @@
           <td><%= taskRoles[taskRole].taskStatuses[i].taskIndex %></td>
           <td><%= taskRoles[taskRole].taskStatuses[i].containerId %></td>
           <td><%= taskRoles[taskRole].taskStatuses[i].containerIp %></td>
+          <td>
+            <% for (let portLabel of Object.keys(taskRoles[taskRole].taskStatuses[i].containerPort)) { %>
+              <p><%= portLabel %>: <%= taskRoles[taskRole].taskStatuses[i].containerPort[portLabel] %></p>
```

Comment:
Better keep the list in one line?

Reply:
It's too long. One port type per line makes it clearer.